src/common/mockFactory.ts

```diff
@@ -438,10 +442,10 @@ export default class MockFactory {
                 height: size.height,
             },
             points: [
-                { x: origin.x, y: origin.y }, // Top left
-                { x: origin.x + size.width, y: origin.y }, // Top Right
-                { x: origin.x, y: origin.y + size.height }, // Bottom Left
-                { x: origin.x + size.width, y: origin.y + size.height }, // Bottom Right
+                {x: origin.x, y: origin.y}, // Top left
+                {x: origin.x + size.width, y: origin.y}, // Top Right
+                {x: origin.x, y: origin.y + size.height}, // Bottom Left
+                {x: origin.x + size.width, y: origin.y + size.height}, // Bottom Right
```

Comment:
Could you make sure we don't put unnecessary changes in this PR? Intended styling change should go to another PR.